lms/tests/conftest.py

```diff
@@ -60,6 +60,7 @@ def webapp_configurations():
     webapp.secret_key = ''.join(
         random.choices(string.ascii_letters + string.digits, k=64),
     )
+    limiter.enabled = False
```

Comment:
How about adding a test that on setup sets `limiter.enabled` to `True`, and in the end sets the teardown to `False`?

We must check that the limiter actually works.

Reply:
How about enabling it in a test, testing it and then disabling it again?

Reply:
Yes, good idea